web/packages/hovercards/src/core.ts

```diff
@@ -506,6 +779,9 @@ export default class Hovercards {
 			// ensuring that the hovercard remains visible when the mouse hovers over the gap
 			hovercard.style[ padding ] = `${ paddingValue }px`;
 
+			// Placing the hovercard at the top-level of the dc to avoid being clipped by overflow
+			dc.body.appendChild( hovercard );
```

Comment:
I have reverted this change and updated the hovercard skeleton to match the new hovercard structure better.
However, because some elements are shown according to the user's preferences, it will be tricky to make the hovercard height 100% accurate.
From my perspective, with the latest changes, the layout shift after the load is acceptable.